core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/internalresponse/EthAccountsResultProvider.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.core.service.http.handlers.internalresponse;
```

Comment:
This is only used by the `requestmapper` so I think this and the classes in `internalresponse` make more sense under a `tech.pegasys.web3signer.core.service.jsonrpc.handlers.internalresponse` package

Reply:
done